src/scikit_package/cli/gh.py

```diff
@@ -49,17 +48,14 @@ def _get_issue_content(issue_url):
     return source_repo_url, issue_content
 
 
-def _broadcast_issue_to_urls(
-    issue_content, broadcast_urls, gh_token, dry_run=False
-):
-    """Broadcast issue to the repos pointed by the urls in the
-    broadcast_urls.
+def _broadcast_issue_to_urls(issue_content, repo_urls, gh_token, dry_run=True):
+    """Broadcast a issue to multiple GitHub repositories.
 
-    Parameters
+     Parameters
     ----------
     issue_content : dict
         The issue_content needed to create issues in each repo.
-    broadcast_urls : list of str
+    repo_urls : list of str
         The urls to the target repos.
     gh_token : str
```

Comment:
do we need to pass this in or is it just set as an environment variable?  I am asking because I am not sure.